### PR TITLE
Added requests for users

### PR DESCRIPTION
**Pull request by zemuvier**
Added post and get requests for working with users

### CODE REVIEW

File: my_dev_client/client.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+from requests import codes
+from requests import delete
+from requests import get
+from requests import exceptions as exc
+from requests import post
+
+URL = 'http://localhost:5000'
```
Comment (IlyaMenkov) [inline my_dev_client/client.py]: did we need this URL?

**Comment by esikachev** [comment]
@zemuvier bingo!

**Comment by zemuvier** [comment]
@esikachev muchas gracias!

File: my_dev_client/client.py (hunk 1 above)
Comment (esikachev) [inline my_dev_client/client.py]: we want to fix it in other patches

File: my_dev_client/runner.py [hunk 2]
```diff
@@ -7,9 +7,11 @@ def main():
     parser = argparse.ArgumentParser(description="My-dev utility.")
     parser.add_argument('command', help="Command for running via my-dev",
                         nargs='*', default=[])
+    parser.add_argument('--host', help="Host to run", nargs='?', default=None)
```
Comment (zemuvier) [inline my_dev_client/runner.py]: Do we need this?

File: my_dev_client/runner.py [hunk 3]
```diff
@@ -7,9 +7,11 @@ def main():
...
 
     args = parser.parse_args()
 
     command_arguments = args.command
+    host_arg = args.host
 
-    os.system(' '.join(command_arguments))
+    os.system(' '.join([command_arguments, host_arg]))
```
Comment (zemuvier) [inline my_dev_client/runner.py]: And this?

File: my_dev_client/client.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+
+class Client(object):
+
+    def __init__(self):
```
Comment (esikachev) [inline my_dev_client/client.py]: let's migrate this variables to methods, like _get(), _post() etc

File: requirements.txt [hunk 5]
```diff
@@ -2,6 +2,8 @@
 # of appearance. Changing the order has an impact on the overall integration
 # process, which may cause wedges in the gate later.
 
+Flask>=0.11.1
```
Comment (zemuvier) [inline requirements.txt]: Do we need Flask here?

File: my_dev_client/runner.py (hunk 2 above)
Comment (esikachev) [inline my_dev_client/runner.py]: i think, we need to add this option in next patches

Comment (zemuvier) [inline my_dev_client/runner.py]: Ok, I will remove it

**Comment by esikachev** [comment]
close #4

**Comment by esikachev** [comment]
@zemuvier please, rename my_de_client to my_dev

**Pull request merged**